.gitignore

```diff
@@ -0,0 +1 @@
+.vscode
```

Comment:
https://www.toptal.com/developers/gitignore/api/linux,macos,windows,visualstudiocode,go

Reply:
fun fact: it does not contains `.vscode` I had to add it manually

Reply:
actually I was wrong, they don't recommend to ignore a few `.vscode` files like tasks, launches, etc. I don't know if it is a good idea but I'll keep that way.

Reply:
Yeah normally the full .vscode folder should not be excluded